rd-net/Lifetimes/Annotations/CallerArgumentExpressionAttribute.cs

```diff
@@ -0,0 +1,14 @@
+﻿// ReSharper disable once CheckNamespace
+namespace System.Runtime.CompilerServices
+{
+  [AttributeUsage(AttributeTargets.Parameter)]
+  internal sealed class CallerArgumentExpressionAttribute : Attribute
```

Comment:
Please, specify license both in the file and in `third-party-notices.txt`.

Reply:
done